### PR TITLE
create mobile-first portflio

### PR DESCRIPTION
**Pull request by MariamOsman11**
# what I did
-Use Figma design to create a UI.
-Flexbox to place elements in the page.
-Build a personal portfolio site.
-Use images and backgrounds to enhance the look of the website.
-Minimum mobile screen size should be 375px
-Minimum desktop screen size should be 768px

### CODE REVIEW

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
Hi @MariamOsman11  🙋‍♂️, 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code

## :no_entry:  Invalid Code Review Request :no_entry: 

- You sent a project without a professional README file and without Linters, kindly add both to proceed with the review :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready :inbox_tray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
Hi @MariamOsman11, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have running linters. Please ensure that your linters are running before you submit for review.

- [ ] Kindly add linters to your project. In the root project, you need to have a folder named `.github`. Inside this one, you need another folder named `workflows`. And inside the `workflows` folder, you need a file named `linters.yml`. Inside this file, you need to have the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
>Here is a video demo I created demostrating  the above:
>
https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <body>
+    
+      <header>
+        <nav>
+          <ul>
+            <li class="logo">MY Logo</li>
+          </ul>
+          <li class="img-logo">
+            <img class="Union-nav" src="images/Union.png" alt="" />
+          </li>
```
Comment (leonardodiasb) [inline index.html:18]: - [ ] The styling of the toolbar is good and similar to the Figma template, but the toolbar doesn't seem to be a list of items so the `ul` and `li` tags would not be appropriate for this case(remember that the `li` tags must be inside the `ul` tag). The logo is usually a link to the home page on most applications. Kindly remove the `ul` and `li` tags and wrap the logo with the `a` tag for a better user experience. Also, the logo should be outside the `nav` tag but inside the `header`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        </nav>
+      </header>
+
+      <section class="main-section">
+        
+        <p>
+          <h3 class="primary-text">I’m Marian
+            Glad to see you!</h3>
+        </p>
```
Comment (leonardodiasb) [inline index.html:27]: - [ ] The page's main title is very similar to the template, but it should be wrapped with the `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. Kindly change that and remove the `p` tag around the title.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @MariamOsman11, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional PR and README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — CHANGES_REQUESTED** [review]
Hi @MariamOsman11 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

> You have submitted a Pull request/project, that does not belong to the mileston... what I mean is you have used the  `setup and mobile first PR` to submitte `finish mobile version milestone`



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

-----
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._ 🟢

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @MariamOsman11 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @MariamOsman11  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional PR👍 
✔️professional README👍 
✔️No Linter Errors👍 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**